tests/functions/test_factorize_columns.py

```diff
@@ -0,0 +1,80 @@
+"""
+Author: Vamsi Krishna
+Date: 23 July 2021
+
+The intent of these tests is to test factorize_columns function works.
```

Comment:
```suggestion
The intent of these tests is to test factorize_columns function works.
Because underneath the hood we are using `pd.factorize`,
we intentionally do not test the values of the resultant dataframe.
That would be duplicating the tests from what the `pandas` library provides.
```

Reply:
Updated the doc string